keps/sig-storage/3141-prevent-volume-mode-conversion/README.md

```diff
@@ -735,6 +735,9 @@ Major milestones might include:
 - when the KEP was retired or superseded
 -->
 
+- 2023-02-06: KEP updated to mark transition to beta
```

Comment:
Can you also add that it was beta off-by-default initially and it moved to beta on-by-default in 1.29?

Reply:
Done